projects/gen-ai.md

```diff
@@ -0,0 +1,157 @@
+# Generative AI Observability
+
+>[!IMPORTANT] This project extends LLM Semantic Conventions project
+
+## Description
+
+Generative AI observability is evolving fast. The industry is still learning how to use GenAI products which creates new opportunities and challenges in the observability space. GenAI observability is focused on development time and verbose telemetry. Non-deterministic nature of GenAI creates the need to measure the quality and safety of responses which become part of observability.
+
+There are multiple GenAI-focused observability projects out there such as [Arize Phoenix](https://docs.arize.com/phoenix), [Langfuse](https://langfuse.com/), [LangSmith](https://www.langchain.com/langsmith), [Langtrace](https://langtrace.ai/), [OpenLit](https://github.com/openlit/openlit), [Traceloop](https://www.traceloop.com/docs/introduction) and many others.
+
+Some of these projects are based on OpenTelemetry, others are "inspired by" it, but don't have out-of-the-box integration with OTel.
+
+The scope of GenAI observability includes instrumentations of various GenAI clients, frameworks, response evaluations, as well as model (server-side) instrumentations.
+
+Example Use Cases:
+
+- **Debugging**: Tracing provides causality for multi-step LLM applications allowing to understand the operation flow and root-cause errors.
+- **Evaluation**: Input/output events record non-deterministic model responses that can be used for debugging or automated evaluations
+  measuring LLM response accuracy and other quality characteristics.
+- **AI Safety**: GenAI telemetry, along with specialized evaluators, can help ensure ethical use of LLMs, maintain privacy compliance, detect bias, and safeguard data security.
+- **Cost Tracking**: Usage metrics based on model self-reporting capabilities provide insights into operational costs.
+- **Performance optimization**: Latency, throughput, and error rate of LLM operations provide traditional performance monitoring layer for
```

Comment:
Can we add more use cases here?

**End-User Experience Tracking**: Monitoring end-user interaction data, such as satisfaction scores and feedback tied to LLM outputs, can provide insights into the quality of model outputs and how they affect user engagement.
**Incident Response Automation**: Integrating observability data with AI operations platforms can enable automated incident responses, such as restarting failing model components or scaling up resources in response to increased demand, without manual intervention.
**Version Control and Rollback**: Tracking model versions in observability data helps monitor the performance impact of model updates, allowing quick rollback to previous versions in case a new release introduces errors or degradations in quality.

Reply:
Added with a caveat:

> End-User Experience Tracking: Monitoring end-user interaction data, such as satisfaction scores and feedback tied to LLM outputs, can provide insights into the quality of model outputs and how they affect user engagement.

I added parts of it to evaluations. It general, the end-user experience is covered by [Client Instrumentation SIG](https://github.com/open-telemetry/community/blob/main/projects/client-instrumentation.md). I believe that on GenAI side we'll need to emit some signals (e.g. https://github.com/open-telemetry/semantic-conventions/pull/1322) and make sure they are compatible and leverage what Client Instrumentation WG does.